src/stan/math/prim/mat/prob/multi_normal_rng.hpp

```diff
@@ -20,10 +20,10 @@ namespace stan {
   namespace prob {
     using Eigen::Dynamic;
 
-    template <class RNG>
+    template <class RNG, typename T1, typename T2>
```

Comment:
But `T1` and `T2` are not scalar types to begin with. You can't constrain the scalar type using the template when you are using `Eigen::MatrixBase<>`